### DISCUSSION TITLE
Is the dashboard delayed?

### DISCUSSION THREAD

**Discussion by batuhannarci** [prisma/optimize-feedback#6] [Q&A]
Everything seems fine but nothing was sent to the dashboard. 

I can see the query logs with the **traceparent** key. 

How can I be sure that are there any errors or not?

**Comment by mykhailo-monchak** [comment]
The same here. It seems to be broken now :(

**Comment by stellan-s** [comment]
The dashboard worked fine for me, however I needed to start recording before running my application (locally).

**Comment by batuhannarci** [comment]
I think that is related to my NestJS setup. Probably extended prisma client could not be used by the system.

**Comment by millsp** [comment]
Hey @batuhannarci @stellan-s @mykhailo-monchak, we have released Prisma Optimize into GA, with many changes, fixes, and new features. Could you please give it a try? https://www.prisma.io/data-platform/optimize
